qmri_vfa/sub-01/fmap/sub-01_acq-tr1_TB1AFI.json

```diff
@@ -0,0 +1,19 @@
+{
+    "IntendedFor": [
+      "fmap/sub-01_TB1map.nii.gz"
+    ],
```

Comment:
Points out of dataset into a derivative. And it looks like `TB1map` is derived from this, so this is used to make, not correct the image, correct?

Reply:
Yes, here `TB1AFI` is `IntendedFor` creating the `fmap/TB1map`. IIRC, it is ok to keep `RawSources` in `fmap/TB1map.json`, but it is not OK to use `IntendedFor` unless it is for correction?

Reply:
Yes, use `RawSources` to point to how you were generated, use `IntendedFor` to link a fieldmap to something that needs correction.